src/bot/events/messageDelete.js

```diff
@@ -10,14 +11,22 @@ export default new EventListener('messageDelete', (message, {client}) => {
 		// Check if message was deleted in a DM or other guilds and do not log if so
 		if (!message.guildID || message.guildID !== config.TEMP_guildID) return;
 
+		if (!message) return;
+
 		// Check if the message is a partial message and if so do not log
 		if (!(message instanceof Eris.Message)) return;
 
 		// Check if the message was from a bot
 		if (message.author.bot) return;
 
+
 		// Ignore if message author has manage message ie. is a mod
-		if (message.channel.permissionsOf(message.author.id).has('manageMessages')) return;
+		try {
+			if (message.channel.permissionsOf(message.author.id).has('manageMessages')) return;
+		} catch (error) {
+			// The line in the try block sometimes crashes when you use the 'Delete Previous Messages' option when manually banning someone. Need to find out why
+			log.error(error);
+		}
```

Comment:
ah fun. I bet it's because we don't get the delete events until the user has already left the server, so there is no member with their ID, so `permissionsOf` throws "unknown member" or something like that.

If that happens, the user who was banned probably isn't a mod, so we can probably continue without doing anything with the caught error.

Reply:
Yeah, removed the log